### PR TITLE
Create LessOrEqual and GreaterOrEqual Ops

### PR DESCRIPTION
**Pull request by zrphercule**
Clean and recreate a pr for LE & GE ops, and also create opset11.

### CODE REVIEW

**Comment by zrphercule** [comment]
@houseroad @linkerzhang @ebarsoum Would you mind take a look again? Thanks!

File: onnx/backend/test/case/model/greater_or_equal.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            nodes=[node],
+            name='GreaterOrEqual',
+            inputs=[onnx.helper.make_tensor_value_info('x',
+                                                       onnx.TensorProto.FLOAT,
+                                                       x.shape),
+                    onnx.helper.make_tensor_value_info('y',
+                                                       onnx.TensorProto.FLOAT,
+                                                       y.shape)],
+            outputs=[onnx.helper.make_tensor_value_info('z',
+                                                        onnx.TensorProto.FLOAT,
```
Comment (houseroad) [inline onnx/backend/test/case/model/greater_or_equal.py]: onnx.TensorProto.BOOL

File: onnx/backend/test/case/model/greater_or_equal.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+class GreaterOrEqual(Base):
+
+    @staticmethod
+    def export():  # type: () -> None
+        node = onnx.helper.make_node(
+            'GreaterOrEqual',
+            ['x', 'y'],
+            ['less_or_equal'],
```
Comment (houseroad) [inline onnx/backend/test/case/model/greater_or_equal.py]: greater_or_equal

File: onnx/backend/test/case/model/less_or_equal.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            nodes=[node],
+            name='LessOrEqual',
+            inputs=[onnx.helper.make_tensor_value_info('x',
+                                                       onnx.TensorProto.FLOAT,
+                                                       x.shape),
+                    onnx.helper.make_tensor_value_info('y',
+                                                       onnx.TensorProto.FLOAT,
+                                                       y.shape)],
+            outputs=[onnx.helper.make_tensor_value_info('z',
+                                                        onnx.TensorProto.FLOAT,
```
Comment (houseroad) [inline onnx/backend/test/case/model/less_or_equal.py]: onnx.TensorProto.BOOL

**Comment by wschin** [comment]
Would you consider making it a FunctionProto? It can be composed using around 6 simple operators. If every such an operator will be added into ONNX, ONNX will end up with infinite amount of signatures.

**Comment by postrational** [comment]
We should consider adding this as a `Function`.

**Pull request closed without merging**